specs/test_formats/sanity/README.md

```diff
@@ -0,0 +1,7 @@
+# Sanity tests
```

Comment:
I'm not certain it is worth calling these "sanity" tests. `blocks` tests are what eth1 calls "blockchain" tests and will likely in the end have the most test cases as they test the entire state transition.

Reply:
maybe just `state_transition` as the dir here and have state transitions triggered by blocks and those triggered by slots. (as is just rename).

The initial blocks tests are the `sanity` suite, but we might have many more blocks suites

Reply:
I think it's nice to have sanity apart from the others, as one may run these before anything else, to catch obvious problems first.
`slots` and `blocks` pretty much direct the format, like a handler should. It's not the intention to put future blockchain tests in there.

We can easily create a `state_transition` runner, with a handler that does the same (and possibly split that up into more descriptive handlers, e.g. `slow_blocks`, `orphan_blocks`, `slashable_blocks` and so on.)

Reply:
works for me